package chapter11_12;

public class Point {

	private int x;
	private int y;

	// x座標、y座標ともに 0で初期化する
	public Point() {
		this.x = 0;
		this.y = 0;
	}
	
	public Point(int x,int y) {
		this.x = x;
		this.y = y;
	}
	
	public int getX() {
		return x;
	}
	
	public void setX(int x) {
		this.x = x;
	}
	
	public int getY() {
		return y;
	}
	
	public void setY(int y) {
		this.y = y;
	}
}
